### PR TITLE
rules update for OCP-20932 OCP-24423

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/overview.xyaml [hunk 1]
```diff
@@ -6,11 +6,11 @@ wait_overview_items_loaded:
 check_list_heading_shown:
   element:
     selector:
-      xpath: //h2[@class='project-overview__group-heading' and text()='<heading>']
+      xpath: //button[contains(@class,'project-overview__item-heading--name') and text()='<heading>']
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/overview.xyaml]: xpath changed

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 2]
```diff
@@ -324,11 +324,11 @@ check_resource_details_key_and_value:
 check_key_and_resource_link:
   element:
     selector:
-      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//a[contains(@href,'<resource_link>')]
+      xpath: //button[(text()='<key>')]/parent::dt/following-sibling::dd[1]//a[contains(@href,'<resource_link>')] | //dt[contains(text(),'<key>')]/following-sibling::dd[1]//a[contains(@href,'<resource_link>')]
 check_key_and_action_link:
   element: &key_action
     selector:
-      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button[text()='<text_in_action>'] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1]//button[text()='<text_in_action>']
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: some key are `//dt` while some are `//button`

**Comment by yapei** [comment]
@yanpzhan  Please help review. thanks

**Comment by yanpzhan** [comment]
LGTM
@akostadinov or @pruan-rht pls help to merge if no comment, thanks!

**Pull request merged**